src/main/java/uk/gov/hmcts/reform/pbis/model/EmailToSend.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.pbis.model;
+
+import java.util.Map;
+
+public final class EmailToSend {
```

Comment:
I've prepared our internal models as immutable classes with no accessors. As this may be controversial for some people, I can add getters. Please let me know if that's the case.